pkg/services/config.go

```diff
@@ -69,6 +69,10 @@ type Config struct {
 	NoReplyEmailAddress        string `usage:"The email to use for no-reply emails from obot"`
 	Docker                     bool   `usage:"Enable Docker support" default:"false" env:"OBOT_DOCKER"`
 
+	// Sendgrid webhook
+	SendgridWebhookUsername string `usage:"The username for the sendgrid webhook to authenticate with" env:"OBOT_SENDGRID_WEBHOOK_USERNAME"`
+	SendgridWebhookPassword string `usage:"The password for the sendgrid webhook to authenticate with" env:"OBOT_SENDGRID_WEBHOOK_PASSWORD"`
```

Comment:
All the other configuration variables we use to configure the server start with `OBOT_SERVER_`. If you remove the `env` tag here, that will be the same for these configuration variables.